public/app/features/dashboard-scene/pages/DashboardScenePageStateManager.ts

```diff
@@ -172,10 +181,10 @@ export class DashboardScenePageStateManager extends StateManagerBase<DashboardSc
     throw new Error('Snapshot not found');
   }
 
-  public async loadDashboard(options: LoadDashboardOptions) {
+  public async loadDashboard(options: LoadDashboardOptions, params?: string) {
```

Comment:
can't find what this params is or where it is passed (where is it called with this params?) why is it a separate function arg and not part of LoadDashboardOptions?

Reply:
`params` are the `queryParams` passed for the reload.

I renamed them to `queryParams` and made them part of `LoadDashboardOptions`